Plugins/GitFlow/GitFlowForm.Designer.cs

```diff
@@ -444,10 +445,20 @@ private void InitializeComponent()
             this.lblRunCommand.AutoSize = true;
             this.lblRunCommand.Location = new System.Drawing.Point(60, 14);
             this.lblRunCommand.Name = "lblRunCommand";
-            this.lblRunCommand.Size = new System.Drawing.Size(11, 13);
+            this.lblRunCommand.Size = new System.Drawing.Size(10, 13);
             this.lblRunCommand.TabIndex = 1;
             this.lblRunCommand.Text = "-";
             // 
+            // cbSquashFeature
+            // 
+            this.cbSquashFeature.AutoSize = true;
+            this.cbSquashFeature.Location = new System.Drawing.Point(492, 113);
+            this.cbSquashFeature.Name = "cbSquashFeature";
+            this.cbSquashFeature.Size = new System.Drawing.Size(98, 17);
+            this.cbSquashFeature.TabIndex = 9;
```

Comment:
Can you please confirm the tab order is logical.
If not, please fix it.

Reply:
Hi @RussKie, I confirm the number 9 as TabIndex for `cbSquashFeature`